scripts/compareAPIs.js

```diff
@@ -2,360 +2,545 @@
 let fs = require('fs-extra');
```

Comment:
Request for future, not this PR - Could this file be split into a couple scripts/files to make this seem more like a program?

Reply:
I don't follow, happy to refactor (or someone else can) down the line if you have a specific idea